package.json

```diff
@@ -72,6 +72,7 @@
     "eslint-config-prettier": "6.15.0",
     "eslint-plugin-import": "2.22.1",
     "eslint-plugin-node": "11.1.0",
+    "eslint-plugin-prettier": "^3.1.4",
```

Comment:
I don't know if using this plugin is a good idea or not, but at least pin the dependency version. IE, remove the `^`.

Reply:
Why? It's locked in yarn.lock and also checked in.

Reply:
I updated PR with pinned version.

To elaborate on what this plugin does, it allows for prettier to run as part of eslint, which means prettier errors can be caught and autofixed by eslint without having to run prettier. (i.e. "eslint --fix" will take care of prettier errors as well)